huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_log_converge_test.go

```diff
@@ -25,48 +25,60 @@ func TestAccLogConverge_basic(t *testing.T) {
 	var (
 		obj   interface{}
 		rName = "huaweicloud_lts_log_converge.test"
+		name  = acceptance.RandomAccResourceName()
 		rc    = acceptance.InitResourceCheck(rName, &obj, getLogConvergeResourceFunc)
 	)
 
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck: func() {
 			acceptance.TestAccPreCheck(t)
 			acceptance.TestAccPreCheckLTSLogConvergeBaseConfig(t)
-			acceptance.TestAccPreCheckLTSLogConvergeGroupConfig(t)
-			acceptance.TestAccPreCheckLTSLogConvergeStreamConfig(t)
+			acceptance.TestAccPreCheckLTSLogConvergeMappingConfig(t)
 		},
 		ProviderFactories: acceptance.TestAccProviderFactories,
 		CheckDestroy:      rc.CheckResourceDestroy(),
 		Steps: []resource.TestStep{
 			{
-				Config: testLogConverge_basic(90),
+				Config: testLogConverge_basic_step1(name),
 				Check: resource.ComposeTestCheckFunc(
 					rc.CheckResourceExists(),
 					resource.TestCheckResourceAttr(rName, "organization_id", acceptance.HW_LTS_LOG_CONVERGE_ORGANIZATION_ID),
 					resource.TestCheckResourceAttr(rName, "management_account_id", acceptance.HW_LTS_LOG_CONVERGE_MANAGEMENT_ACCOUNT_ID),
 					resource.TestCheckResourceAttr(rName, "member_account_id", acceptance.HW_LTS_LOG_CONVERGE_MEMBER_ACCOUNT_ID),
 					resource.TestCheckResourceAttr(rName, "log_mapping_config.0.source_log_group_id",
 						acceptance.HW_LTS_LOG_CONVERGE_SOURCE_LOG_GROUP_ID),
-					resource.TestCheckResourceAttr(rName, "log_mapping_config.0.target_log_group_name",
-						acceptance.HW_LTS_LOG_CONVERGE_TARGET_LOG_GROUP_NAME),
-					resource.TestCheckResourceAttr(rName, "log_mapping_config.0.target_log_group_id",
-						acceptance.HW_LTS_LOG_CONVERGE_TARGET_LOG_GROUP_ID),
+					resource.TestCheckResourceAttr(rName, "log_mapping_config.0.target_log_group_name", name),
+					resource.TestCheckResourceAttrSet(rName, "log_mapping_config.0.target_log_group_id"),
+					resource.TestCheckResourceAttr(rName, "log_mapping_config.0.log_stream_config.#", "0"),
+				),
+			},
+			{
+				Config: testLogConverge_basic_step2(name, 90),
```

Comment:
It looks like the second parameter doesn't need to exist.
Same to function `testLogConverge_basic_step3`

Reply:
sure